package.json

```diff
@@ -24,7 +24,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "1.54.0-insider"
+        "vscode": "1.55.0-insider"
```

Comment:
Should this have the ```^```?

Reply:
Not sure we want that,
we want this to be only compatible with 1.55.0-insiders & nothing else.
E.g. if user has `1.55.0` stable, that should not be used.
The API is currently locked into a specific version of VS Code.

Reply:
Also we dno't want insider users in `1.56.0-insiders` to get this VSIX.

Reply:
Let me know what you think about that.